### PR TITLE
ROX-23078: Add sorting to Cluster single page table

### PR DESCRIPTION
**Pull request by dvail**
## Description

As titled.

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

## Testing Performed

Visit a Cluster single page, and verify that the default sort is by CVSS, descending:
image
Sort ascending:
image
CVE descending
image
CVE ascending
image
CVE status descending
image
CVE status ascending
image
CVE type descending
image

CVE type ascending
image

### CODE REVIEW

File: ui/apps/platform/src/Containers/Vulnerabilities/PlatformCves/Cluster/CVEsTable.tsx [hunk 1]
```diff
@@ -26,6 +27,15 @@ function displayCveType(cveType: string): string {
     }
 }
 
+export const sortFields = {
+    CVE: 'CVE',
+    CVEStatus: 'Cluster CVE Fixable',
+    CVEType: 'CVE Type',
+    CVSSScore: 'CVSS',
+};
```
Comment (dvail) [inline ui/apps/platform/src/Containers/Vulnerabilities/PlatformCves/Cluster/CVEsTable.tsx:35]: CVE Status field name taken from https://github.com/stackrox/stackrox/blob/master/pkg/search/options.go#L95
CVE Type field name taken from https://github.com/stackrox/stackrox/blob/master/pkg/search/options.go#L54

All of these were tested against staging, where there is more data, but unfortunately still only a single `Type` of CVE across cluster so I was unable to validate the data when sorting by `CVEType`.

File: ui/apps/platform/src/Containers/Vulnerabilities/PlatformCves/PlatformCve/AffectedClustersTable.tsx [hunk 2]
```diff
@@ -14,7 +14,7 @@ import { displayClusterType } from '../utils/stringUtils';
 
 export const sortFields = {
     Cluster: 'Cluster',
-    ClusterType: 'Cluster Type',
+    ClusterType: 'Cluster Platform Type',
```
Comment (dvail) [inline ui/apps/platform/src/Containers/Vulnerabilities/PlatformCves/PlatformCve/AffectedClustersTable.tsx:17]: Just discovered this was [not the correct field](https://github.com/stackrox/stackrox/pull/10844/files#diff-4e9e719f0bb6eaeccc0a709142b9a556570bccf8b869de2d33da8147bb052444R146), so I'm piggybacking this change on a related sorting PR...

File: ui/apps/platform/src/Containers/Vulnerabilities/PlatformCves/Cluster/CVEsTable.tsx (hunk 1 above)
Comment (pedrottimark) [reply ui/apps/platform/src/Containers/Vulnerabilities/PlatformCves/Cluster/CVEsTable.tsx:35]: A thought, not necessarily a change request. Based on discussion at office hours, what is pro and con:

```ts
export const CVE_SORT_FIELD = 'CVE';
export const CVE_STATUS_SORT_FIELD = 'Cluster CVE Fixable';
export const CVE_TYPE_SORT_FIELD = 'CVE Type';
export const CVSS_SCORE_SORT_FIELD = 'CVSS';
```

**Review by pedrottimark — APPROVED** [review]

File: ui/apps/platform/src/Containers/Vulnerabilities/PlatformCves/Cluster/CVEsTable.tsx (hunk 1 above)
Comment (dvail) [reply ui/apps/platform/src/Containers/Vulnerabilities/PlatformCves/Cluster/CVEsTable.tsx:35]: ➕ I'm working on a Container-centralized sort options export file, complete with SCREAMING SNAKES as discussed at our meeting on Monday.

**Comment by dvail** [comment]
ocp-4-15 failure due to known flake: `Configuration Management Secrets`

**Pull request merged**